test/api/pipelineUsage.test.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const { assert } = require('chai');
+const { api } = require('../..');
+const { validate } = require('../helper');
+
+describe('api get pipelineUsage', () => {
+    const inputSchema = api.pipelineUsage.get;
```

Comment:
This is not input schema
```suggestion
    const schema = api.pipelineUsage.get;
```